contracts/router/interfaces/traderjoe/ILBPair.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {IERC20} from "@openzeppelin/contracts-4.8.0/token/ERC20/IERC20.sol";
```

Comment:
We're not using any functions from `IERC20`, it's only used for casting to comply with the defined interface. 

Since both `IERC20` and `address` are encoded in the same way (same goes for `IERC20[]` and `address[]`), we could use `address` instead of `IERC20` in all the interfaces, which would increase the readibility of the code w/o losing any functionality.